app/controllers/watchers_controller.rb

```diff
@@ -105,20 +105,7 @@ def set_watcher(user, watching)
 
     respond_to do |format|
       format.html { redirect_to :back }
-      format.js do
-        if params[:replace].present?
-          if params[:replace].is_a? Array
-            @replace_selectors = params[:replace]
-          else
-            @replace_selectors = params[:replace].split(',').map(&:strip)
-          end
-        else
-          @replace_selectors = ['#watcher']
-        end
-        @user = user
-
-        render action: 'replace_selectors'
-      end
+      format.js
```

Comment:
If you remove the `@replace_selectors` variable here, it should also be removed in the rjs template.